src/import_edf.jl

```diff
@@ -299,13 +344,20 @@ following transformations:
 See the OndaEDF README for additional details regarding EDF formatting expectations.
 """
 function store_edf_as_onda(path, edf::EDF.File, uuid::UUID=uuid4();
-                           custom_extractors=(), import_annotations::Bool=true,
+                           custom_extractors=STANDARD_EXTRACTORS, import_annotations::Bool=true,
                            signals_prefix="edf", annotations_prefix=signals_prefix)
     EDF.read!(edf)
     file_format = "lpcm.zst"
 
     signals = Any[]
-    edf_samples = edf_to_onda_samples(edf; custom_extractors=custom_extractors)
+    edf_samples, errors = edf_to_onda_samples(edf; custom_extractors=custom_extractors)
+    for error in errors
+        if isa(e, SamplesInfoError)
+            @warn e.msg exception=e.cause
+        else
+            @warn exception=e
```

Comment:
If it's really critical that these kinds of errors don't actually cause the job to crash/fail then they could use tests. (and same for above with the `SamplesInfoError`)

Reply:
The test that takes in `test/test_edf_to_samples_info.out` is fairly thorough, though it's not systematic, just has a lot of varied test cases...